tests/software_tests/can/packet/test_can_packet.py

```
@@ -0,0 +1,406 @@
+          "st_min": None}),
+        # FF
+        ({"packet_type": CanPacketType.FIRST_FRAME,
+          "addressing_format": CanAddressingFormat.NORMAL_FIXED_ADDRESSING,
+          "addressing_type": AddressingType.FUNCTIONAL,
+          "can_id": 0x18DB023B,
+          "target_address": 0x02,
+          "source_address": 0x3B,
+          "dlc": 0xF,
+          "payload": bytes(range(50, 108)),
+          "data_length": 0xFEDCBA98},
+         {"raw_frame_data": bytes([0x10, 0x00, 0xFE, 0xDC, 0xBA, 0x98] + list(range(50, 108))),
```

Ruff (0.12.2)

Consider `[16, 0, 254, 220, 186, 152, *list(range(50, 108))]` instead of concatenation

Replace with `[16, 0, 254, 220, 186, 152, *list(range(50, 108))]`

(RUF005)